src/main/java/org/opentripplanner/model/plan/StreetLeg.java

```diff
@@ -127,9 +127,16 @@ public LineString getLegGeometry() {
     return legGeometry;
   }
 
+  List<P2<Double>> getUnRoundedLegElevation() {
```

Comment:
I'm not sure I like "unrounded". How about "getFullLegElevation" or "getPrecisesLegElevation"?